### ISSUE TITLE
Training Loss Fails to Converge in train_ground.py？

### ISSUE THREAD

**Issue by M21F-Liao** [hyzhou404/HUGSIM#36]
Thank you for your excellent work! While reproducing this project, I noticed that several train_loss values in train_ground.py **failed to converge**. 
What could be the reasons for this issue? (I’m using the sample dataset Nuscenes-0383, default hyperparameters）

<img width="1489" height="481" alt="Image" src="https://github.com/user-attachments/assets/3224a703-9ad5-49a4-8052-33060ac70809" />

**Comment by hyzhou404** [comment]
Hi, I have not encountered this situation during my experiments, particularly with the sample data nusc-0383, which has been thoroughly tested. Could you please provide more details, such as rendered images and any changes made to the code?

**Comment by hyzhou404** [comment]
The results appear to be correct, and the PSNR is infinite due to a numerical error in the evaluation. I have also encountered numerical errors in my experiments, but they did not impact the overall results. I think you can proceed to the next step!

**Comment by M21F-Liao** [comment]
> The results appear to be correct, and the PSNR is infinite due to a numerical error in the evaluation. I have also encountered numerical errors in my experiments, but they did not impact the overall results. I think you can proceed to the next step!

I think the raw input point cloud data(ground_points3d.ply) was already close enough to the ground truth, resulting in a very small initial loss, which is why the optimization process didn't have a noticeable effect. Thank you for your feedback!
